lessons/pydata/ml/tmp/Druhá hodina.ipynb

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
<p>To pak asi taky zmizí, že jo...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/77/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>